config/index.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+connect_app_descriptor = {
+    "name": "Sample Connect App Python",
+    "description": "Atlassian Connect app - Python",
+    "key": "com.atlassian.sample-app-python",
+    "baseUrl": "https://6333-180-150-77-245.ngrok.io",
```

Comment:
Hard coding this for now. I'll open a separate ticket for us to tackle updating this programmatically.